client/src/components/Bio/Bio.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import "./Bio.css";
+import Icon from "../Icon/Icon";
+import { Link } from "react-router-dom";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faEdit } from "@fortawesome/free-solid-svg-icons";
```

Comment:
If you're using the `Icon` component, you shouldn't have to do this here. Can you move this to `App.js`?